src/services/Database/IndexedDB/IndexedDB.js

```diff
@@ -150,6 +150,101 @@ const deleteAccount = async (accountId, onError, DB = IDB) => {
   }
 }
 
+const saveDbToJSON = async (onError, DB = IDB) => {
```

Comment:
If this function intended for full backup we need to add somewhere button for it, or that is required in next phase?